example_app/web/templates/page/index.html.eex

```diff
@@ -37,3 +54,9 @@
     </ul>
   </div>
 </div>
+
+<script type='text/javascript'>
+  $('.logout-link').click(function() {
+    $.ajax({url:"/logout", method: "DELETE"}).done(function() { window.location.reload(); });
+  });
```

Comment:
That makes sense, I didn't realize that was the case. I should look at the example app a little closer.